Application/EdFi.Ods.Admin.Api.UnitTests/Api/OdsApiValidatorTests.cs

```diff
@@ -4,40 +4,40 @@
 // See the LICENSE and NOTICES files in the project root for more information.
 
 using System.Threading.Tasks;
-using EdFi.Ods.AdminApp.Management.Api;
-using EdFi.Ods.AdminApp.Management.Services;
+using EdFi.Ods.Admin.Api.Infrastructure.Api;
+using EdFi.Ods.Admin.Api.Infrastructure.Services;
 using NUnit.Framework;
 using Shouldly;
 
-namespace EdFi.Ods.AdminApp.Management.UnitTests.Api
+namespace EdFi.Ods.Admin.Api.UnitTests.Api;
+
+[TestFixture]
+public class OdsApiValidatorTests
```

Comment:
Except for fixing namespace references and some whitespace differences, I did not make any changes to the code. All of the files now have the namespace listed at the top with a semi-colon at the end of the line, which means the rest of the file does not needed to be wrapped in a set of curly braces. This reduces the indentation and makes it easier to read longer lines of code. Unfortunately, it is also has the effect of making this code review a little confusing because GitHub's color coding does not recognize that the only differences (other than namespace renaming) between this file and the prior version are a semi-colon on line 12, removal of `{` and `}` for the namespace, and reduction of whitespaces on every line.